src/features/presence/PresenceAvatarList.tsx

```diff
@@ -63,39 +73,56 @@ function PresenceAvatarList(props: Props, ref: React.Ref<HTMLDivElement>): JSX.E
         return null;
     }
 
-    return (
+    const renderAvatar = (collaborator: Collaborator) => {
+        const { id, avatarUrl, name, isActive, interactedAt, interactionType } = collaborator;
+
+        const avatarElement = (
+            <PresenceAvatar
+                aria-hidden="true"
+                avatarUrl={avatarUrl}
+                id={id}
+                isActive={isActive}
+                name={name}
+                onBlur={hideTooltip}
+                onFocus={() => showTooltip(id)}
+                onMouseEnter={() => showTooltip(id)}
+                onMouseLeave={hideTooltip}
+                {...avatarAttributes}
+            />
+        );
+
+        const tooltipContent = (
+            <PresenceAvatarTooltipContent
+                name={name}
+                interactedAt={interactedAt}
+                interactionType={interactionType}
+                isActive={isActive}
+            />
+        );
+
+        if (isPreviewModernizationEnabled) {
+            return (
+                <BPTooltip key={id} content={tooltipContent} side="bottom" hidden={hideTooltips}>
```

Comment:
I believe the pattern we usually use for hiding tooltips is not wrapping the component in a Tooltip if we want to hide them